sdk/spring-sdk/src/main/java/kalix/springsdk/annotations/Subscribe.java

```diff
@@ -28,14 +28,8 @@
    * Annotation for subscribing to updates from a Value Entity. It can be used both at type and
    * method levels. When used at type level, it means the `View` will not be transforming state.
    * When used at method level, it gives the ability to transform the updates into a different state
-   * for the view and the underlying method must be declared to receive one or two parameters:
-   *
-   * <ul>
-   *   <li>when one parameter is passed, the single parameter will be considered the event type such
-   *       method will handle;
-   *   <li>when two parameters are passed, the first one will be considered the view state and the
-   *       second one the event type.
-   * </ul>
+   * for the view or delete the view state (handleDeletes flag).
+   * The underlying method must be declared to receive zero (handle deletes) or one (handle update) parameters.
```

Comment:
I feel that the last sentence is not needed. It's already explained on the `handleDeletes` field.